docs/build-the-docs.md

```diff
@@ -49,6 +66,8 @@ Click "**server/**", and you'll see the local copy of the documentation.
 ![Viewing the locally generated Antora documentation](./images/viewing-the-locally-generated-antora-documentation.png)
 
 If you're happy with your changes, as with any other change, create a set of meaningful commits and push them to the remote repository.
+If you're _not_ happy with the changes, however, continue to make further updates, as necessary, and run `antora site.yml` afterward.
```

Comment:
@mmattel, does this answer [your question](https://github.com/owncloud/docs/pull/110#issuecomment-423449654)?

Reply:
My own wording can be improved: ``happy`` --> ``satisfied`` 😄 
``afterward`` --> ``afterwards`` ?